filetests/isa/x86/binary64.cton

```diff
@@ -594,6 +594,75 @@ ebb0:
     [-,%r8]              v520 = ushr_imm v4, 63   ; bin: 49 c1 e8 3f
 
 
+    ; Load Complex
+    [-,%rax]            v521 = iconst.i64 1
+    [-,%rbx]            v522 = iconst.i64 1
+    [-,%rdi]            v523 = iconst.i32 1
+    [-,%rsi]            v524 = iconst.i32 1
+    ; asm: movq (%rax,%rbx,1), %rcx
+    [-,%rcx]            v525 = load_complex.i64 v521+v522         ; bin: heap_oob 48 8b 0c 18
+    ; asm: movl (%rax,%rbx,1), %ecx
+    [-,%rcx]            v526 = load_complex.i32 v521+v522         ; bin: heap_oob 8b 0c 18
+    ; asm: movq 1(%rax,%rbx,1), %rcx
+    [-,%rcx]            v527 = load_complex.i64 v521+v522+1       ; bin: heap_oob 48 8b 4c 18 01
+    ; asm: movl 1(%rax,%rbx,1), %ecx
+    [-,%rcx]            v528 = load_complex.i32 v521+v522+1       ; bin: heap_oob 8b 4c 18 01
+
+    ; asm: mov    0x100000(%rax,%rbx,1),%rcx
+    [-,%rcx]            v529 = load_complex.i64 v521+v522+0x1000  ; bin: heap_oob 48 8b 8c 18 00001000
+    ; asm: mov    0x100000(%rax,%rbx,1),%ecx
+    [-,%rcx]            v530 = load_complex.i32 v521+v522+0x1000  ; bin: heap_oob 8b 8c 18 00001000
+    ; asm: movzbq (%rax,%rbx,1),%rcx
+    [-,%rcx]            v531 = uload8_complex.i64 v521+v522         ; bin: heap_oob 48 0f b6 0c 18
+    ; asm: movzbl (%rax,%rbx,1),%ecx
+    [-,%rcx]            v532 = uload8_complex.i32 v521+v522         ; bin: heap_oob 0f b6 0c 18
+    ; asm: movsbq (%rax,%rbx,1),%rcx
+    [-,%rcx]            v533 = sload8_complex.i64 v521+v522         ; bin: heap_oob 48 0f be 0c 18
+    ; asm: movsbl (%rax,%rbx,1),%ecx
+    [-,%rcx]            v534 = sload8_complex.i32 v521+v522         ; bin: heap_oob 0f be 0c 18
+    ; asm: movzwq (%rax,%rbx,1),%rcx
+    [-,%rcx]            v535 = uload16_complex.i64 v521+v522         ; bin: heap_oob 48 0f b7 0c 18
```

Comment:
Please vertically line up the comments in the lines above. It doesn't have to match the rest of the file or anything, just make it so that everything grouped together here is consistent.